src/views/tables/Orderbook.tsx

```diff
@@ -59,24 +59,23 @@ const useCalculateOrderbookData = ({ maxRowsPerSide }: { maxRowsPerSide: number
   return useMemo(() => {
     const asks = (orderbook?.asks?.toArray() ?? [])
       .map(
-        (row: OrderbookLine, idx: number) =>
-          ({
+        (row: OrderbookLine, idx: number): RowData =>
+          Object.assign({}, row, {
             key: `ask-${idx}`,
-            side: 'ask',
+            side: 'ask' as const,
```

Comment:
we weren't spreading row here and typescript couldn't tell us because of the cast. This was breaking mobile orderbook